addons/osfstorage/tests/utils.py

```diff
@@ -63,3 +64,26 @@ def recursively_create_folder(settings, path):
     for subpath in path:
         current = current.append_folder(subpath)
     return current.append_file(final)
+
+
+def make_payload(user, name, **kwargs):
```

Comment:
It's also used in [Quickfiles tests](https://github.com/CenterForOpenScience/osf.io/pull/7339/files#diff-46cd1bab6ae49e96cd33eb4457cce3dbR18), I figured it was better to abstract this method to utils instead of duplicating the exact same code in two different places